src/Board.js

```diff
@@ -1,76 +1,76 @@
 //parent component that will hold the cards
-import React, { Component } from 'react'
-import './Board.css'
-import Category from './Category'
-import Modal from './Modal'
+import React, { Component } from "react"
+import "./Board.css"
+import Category from "./Category"
+import Modal from "./Modal"
 
 class Board extends Component {
   constructor() {
     super()
     this.state = {
       displayModal: false,
-      categories:[
+      categories: [
         {
           name: "CANADA",
           questions: [
             {
-              price: 10,
+              price: "$200",
               question:
                 "How many provinces and territories are there in Canada?",
               answer: "Canada has ten provinces and three territories."
             },
             {
-              price: 20,
+              price: "$400",
               question: "How many time zones are there in Canada?",
               answer: "Canada has six time zones."
             },
             {
-              price: 30,
+              price: "$600",
               question:
                 "True/False: Canada is the third largest country in the world?",
               answer:
-                "False. Canada is the second largest country in the world."
+                "False. Ca  nada is the second largest country in the world."
             },
             {
-              price: 40,
+              price: "$800",
```

Comment:
having price as literal string doesn't really make sense as the scores need to be added for each player, and it'd be annoying to parse int all, and also removing the dollar sign. we can keep price as int and add dollar sign in Card component as a string

Reply:
Sure that makes sense